src/components/ui/Notification/Notification.jsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect } from "react";
+import { toast, Toaster } from "react-hot-toast";
+import "./Notification.css";
+
+const Notification = ({ isCorrect, isTimeout, explanation, handleNextQuestion }) => {
+  const toastColor = isTimeout
+    ? "toast-incorrect"
+    : isCorrect
+    ? "toast-correct"
+    : "toast-incorrect";
+
+  const title = isTimeout
+    ? "Time Out"
+    : isCorrect
+    ? "Yes! Right Answer"
+    : "OOPS! Wrong Answer";
+
+  useEffect(() => {
+    const toastId = toast.custom(
```

Comment:
this shouldnt be in a use effect. We can achieve our logic without useeffects. We have too many dependencies which will result in bugs.